### PR TITLE
Add initial structure for events

### PR DESCRIPTION
**Pull request by robingrindrod**
I've added in an initial structure for the events. The events can be either listed on the homepage or on a dedicated page.

### CODE REVIEW

**Comment by robingrindrod** [comment]
I've added it to the homepage
image

**Comment by robingrindrod** [comment]
This is the detail page:

image

**Comment by erinsweater** [comment]
Looks good to me. 
Could we change the button to say one of the following: 
Get tickets
More info 
Book now

File: jekyll_files/_layouts/event.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<dl>
+    {% if page.start_time %}<dt>Start time</dt><dd>{{ page.start_time | date: "%e %B at %H:%M" }}</dd>{% endif %}
+    {% if page.location %}<dt>Location</dt><dd>{{ page.location }}</dd>{% endif %}
+</dl>
+
+{{ content }}
+
+{% if page.register_url %}
+<div class="button">
+<a href="{{ page.register_url }}">Register</a>
```
Comment (robingrindrod) [inline jekyll_files/_layouts/event.html:16]:
```suggestion
<a href="{{ page.register_url }}">Book now</a>
```

**Pull request merged**